examples/jdbc2-init-options/jdbc2_init_options.bal

```diff
@@ -0,0 +1,72 @@
+import ballerina/io;
+import ballerina/java.jdbc;
+import ballerina/sql;
+
+function initializeClients() returns sql:Error? {
+    // Simple JDBC Client for H2 database with the mandatory
+    // field - JDBC url. This client can be used with any database
+    // by providing the corresponding JDBC URL and placing the relevant database
+    // driver jar.
+    jdbc:Client jdbcClient1 = check new ("jdbc:h2:file:./target/sample1");
+    io:println("Simple JDBC client created.");
+
+    // Initialize the JDBC client along with username and password.
+    jdbc:Client jdbcClient2 = check new ("jdbc:h2:file:./target/sample2", "rootUser", "rootPass");
+    io:println("JDBC client with user/password created.");
+
+    // Initialize the JDBC client along with some additional
+    // database properties. The database properties can differ based
+    // on the specific JDBC datasource implementation of the database.
+    jdbc:Options h2Options = {
+        datasourceName: "org.h2.jdbcx.JdbcDataSource",
+        properties: {"loginTimeout": "2000"}
+    };
+    jdbc:Client jdbcClient3 = check new ("jdbc:h2:file:./target/sample3", "rootUser", "rootPass", h2Options);
```

Comment:
Shall we wrap this at 80 columns? Otherwise this will cause problems when rendering the example. Refer to https://github.com/ballerina-platform/ballerina-lang/blob/master/docs/bbe-guidelines/bbe-guidelines.md for the practices followed when writing BBEs